src/extension.ts

```diff
@@ -96,7 +96,7 @@ export function activate(context: vscode.ExtensionContext) {
           // we can't use the "includes" and "excludes" patterns from the config file because we
           // want to ensure consistent path matching behaviour... so don't want to rely on vscode's
           // pattern matching being the same.
-          const pattern = new vscode.RelativePattern(folderInfo.uri, `**/*`);
+          const pattern = new vscode.RelativePattern(folderInfo.uri.path, `**/*`);
```

Comment:
> RelativePattern doesn't take Uri as an argument

I believe it does: https://code.visualstudio.com/api/references/vscode-api#RelativePattern

I think this change won't work on windows (since `.path` will return the url path and not the file system path)

Reply:
my bad then, linter was throwing error here that types are mismatched

Reply:
I reverted this change, since it does seem like it supports it, I'm not sure why it flagged it as a type mismatch before